lib/resource/index.js

```diff
@@ -39,9 +41,19 @@ var resourceAjaxRequest;
 var resourceDoing;
 var resourceAborting = false;
 
+function toggleSync(element) {
+  if (window.stopThatSync === false) {
+    window.stopThatSync = true;
+    element.innerHTML = "unfreeze position";
```

Comment:
@teusbenschop I wasn't sure if there was a way to localize strings coming from JS files.